Lib/logging/handlers.py

```diff
@@ -1548,6 +1561,8 @@ def start(self):
         This starts up a background thread to monitor the queue for
         LogRecords to process.
         """
+        if self._thread is not None:
+            raise RuntimeError("Listener already started")
```

Comment:
Just to be sure but do we capitalize the exception messages or should have "listener already started" instead?

Reply:
It seems to sort of vary around. I searched for 
```
raise RuntimeError
```
and found it kind of goes both ways even in the stdlib, though it leans more towards lowercase; i'll update.

Reply:
I was thinking in the logging library mainly